RecoTauTag/RecoTau/plugins/DeepTauId.cc

```diff
@@ -414,6 +414,218 @@ namespace {
+    const reco::PFTauDiscriminator& neutralIsoPtSumWeightCollection;
+    const reco::PFTauDiscriminator& neutralIsoPtSumWeightdR03Collection;
+    const reco::PFTauDiscriminator& photonPtSumOutsideSignalConeCollection;
+    const reco::PFTauDiscriminator& photonPtSumOutsideSignalConedR03Collection;
+    edm::Handle<edm::AssociationVector<reco::PFTauRefProd, std::vector<reco::PFTauTransverseImpactParameterRef>>> PFTauTransverseImpactParameters;
+
+    const float getChargedIsoPtSum(const reco::PFTau& tau, size_t tau_index) {return chargedIsoPtSumCollection.value(tau_index);}
+    const float getChargedIsoPtSum(const pat::Tau& tau, size_t tau_index)  {return tau.tauID("chargedIsoPtSum");}
+
+    const float getChargedIsoPtSumdR03(const reco::PFTau& tau, size_t tau_index) {  return chargedIsoPtSumdR03Collection.value(tau_index);}
+    const float getChargedIsoPtSumdR03(const pat::Tau& tau, size_t tau_index) {  return tau.tauID("chargedIsoPtSumdR03");}
+
+    const float getFootprintCorrectiondR03(const reco::PFTau& tau, size_t tau_index) {  return footprintCorrectionCollection.value(tau_index);}
+    const float getFootprintCorrectiondR03(const pat::Tau& tau, size_t tau_index) {     return tau.tauID("footprintCorrectiondR03");}
+
+
+    const float getNeutralIsoPtSum(const reco::PFTau& tau, size_t tau_index) {  return neutralIsoPtSumCollection.value(tau_index);}
+    const float getNeutralIsoPtSum(const pat::Tau& tau, size_t tau_index) {  return tau.tauID("neutralIsoPtSum");}
+
+    const float getNeutralIsoPtSumdR03(const reco::PFTau& tau, size_t tau_index) {  return neutralIsoPtSumdR03Collection.value(tau_index);}
+    const float getNeutralIsoPtSumdR03(const pat::Tau& tau, size_t tau_index)   {return tau.tauID("neutralIsoPtSumdR03");}
+
+
+    const float getNeutralIsoPtSumWeight(const reco::PFTau& tau, size_t tau_index) { return neutralIsoPtSumWeightCollection.value(tau_index);}
+    const float getNeutralIsoPtSumWeight(const pat::Tau& tau, size_t tau_index) {return tau.tauID("neutralIsoPtSumWeight");}
+
+
+    const float getNeutralIsoPtSumdR03Weight(const reco::PFTau& tau, size_t tau_index) { return neutralIsoPtSumWeightdR03Collection.value(tau_index);}
+    const float getNeutralIsoPtSumdR03Weight(const pat::Tau& tau, size_t tau_index) {return tau.tauID("neutralIsoPtSumWeightdR03");}
+
+
+    const float getPhotonPtSumOutsideSignalCone(const reco::PFTau& tau, size_t tau_index) { return photonPtSumOutsideSignalConeCollection.value(tau_index);}
+    const float getPhotonPtSumOutsideSignalCone(const pat::Tau& tau, size_t tau_index) {return tau.tauID("photonPtSumOutsideSignalCone");}
+
+
+    const float getPhotonPtSumOutsideSignalConedR03(const reco::PFTau& tau, size_t tau_index) {  return photonPtSumOutsideSignalConedR03Collection.value(tau_index);}
+    const float getPhotonPtSumOutsideSignalConedR03(const pat::Tau& tau, size_t tau_index) {return tau.tauID("photonPtSumOutsideSignalConedR03");}
+
+
```

Comment:
Is there a pattern behind the number of empty lines? Please explain or make it more consistent.

Reply:
This was initially to make it more clear to myself what functions were already available but I forgot to remove the empty lines afterwards. This has been done now.